### PR TITLE
s/v2/main/ and other similar changes where applicable

### PR DESCRIPTION
**Pull request by krancour**
This PR straightens out links that break as a result of v2 being pushed onto the main branch (and the existing v2 branch presumably not being long for this world now).

### CODE REVIEW

File: docs/themes/techdoc-brigade/layouts/partials/header.html
Comment (vdice) [inline docs/themes/techdoc-brigade/layouts/partials/header.html:2]: A reminder to update Netlify to deploy v1 docs to this URL.  At time of writing, it doesn't appear to be up yet.

File: README.md
Comment (vdice) [inline README.md:26]: I wonder if we want to make a few additional updates to this README, including (potentially):

 * Revising/rewording `While comprehensive documentation remains a work in progress` (Naturally, more docs are always helpful, but I think we're at a state where most docs are in and updated.
 * Revising/rewording `our quickstart is well-polished and we refer users wishing to test drive Brigade 2 to that documentation.` -- Do we drop '2' now that it is the de facto default?  Defer to your judgement here... I suppose it could also just stay the same/similar...
 * A link/mention to one or both of the v1 GH tree and v1 docs site now that they aren't the defaults

**Review by vdice — COMMENTED** [review]
Looking great!  Only a few thoughts/suggestions on some README updates...

File: docs/themes/techdoc-brigade/layouts/partials/header.html
Comment (krancour) [reply docs/themes/techdoc-brigade/layouts/partials/header.html:2]: Thanks for the reminder. I will re-enable and force push so previews of these changes get built.

File: README.md
Comment (krancour) [reply README.md:26]: For sure. I can include that in this PR. I think there will be some follow-ups to this PR as well if/when we identify more spots like this where we talk about v2 as something new/beta, under "heavy development," etc.

File: docs/themes/techdoc-brigade/layouts/partials/header.html
Comment (krancour) [reply docs/themes/techdoc-brigade/layouts/partials/header.html:2]: Re-enabled.

File: README.md
Comment (krancour) [reply README.md]: I think I've applied all this feedback now. wdyt?

File: docs/themes/techdoc-brigade/layouts/partials/header.html
Comment (vdice) [reply docs/themes/techdoc-brigade/layouts/partials/header.html:2]: Can you double-check this?  Not blocking this PR as it's a Netlify task, but I'm seeing 'Not Found - Request ID: <request id>' when I go to https://v1--brigade-docs.netlify.app/

File: README.md
Comment (vdice) [inline README.md:29]: nit: should it be `makes it easy`?

**Review by vdice — APPROVED** [review]
Updates to the README look great.  Only non-blocking comments.  LGTM!

File: README.md
Comment (krancour) [reply README.md:29]: Yerp

**Pull request merged**
